L1Trigger/L1TCommon/python/customsPostLS1.py

```diff
@@ -78,9 +78,19 @@ def customiseSimL1EmulatorForStage1(process):
 
 from L1Trigger.Configuration.customise_overwriteL1Menu import *
 
+def customiseSimL1EmulatorForPostLS1_lowPU(process):
+    # move to the lowPU v2 L1 menu once the HLT has been updated accordingly
+    process = L1Menu_Collisions2015_lowPU_v2(process)
+    return process
+
 def customiseSimL1EmulatorForPostLS1_50ns(process):
-    # move to the 50ns v0 L1 menu once the HLT has been updated accordingly
-    process = L1Menu_Collisions2015_50ns_v1(process)
+    # move to the 50ns v2 L1 menu once the HLT has been updated accordingly
+    process = L1Menu_Collisions2015_50ns_v2(process)
```

Comment:
Yes, I know it shouldn't have changed anything, as the new seeds are  
never accessed by the paths in the frozen menus used by the MC  
production.

Anyhow, try to stay on the safe side and avoid reasons to delay  
further the inclusion of this PR.

For the same reason, I just reverted #9138, as it was requested by David.

Hope we are done now...
Andrea

Andrea Bocci <notifications@github.com> ha scritto:

>>  def customiseSimL1EmulatorForPostLS1_50ns(process):
>> -    # move to the 50ns v0 L1 menu once the HLT has been updated accordingly
>> -    process = L1Menu_Collisions2015_50ns_v1(process)
>> +    # move to the 50ns v2 L1 menu once the HLT has been updated accordingly
>> +    process = L1Menu_Collisions2015_50ns_v2(process)
>
> Ah, I see that it has been reverted anyway.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/cms-sw/cmssw/pull/9292/files#r31208278